internal/convert.go

```diff
@@ -33,6 +33,7 @@ type Conv struct {
 	ToSource       map[string]NameAndCols              // Maps from Spanner table name to source-DB table name and column mapping.
 	UsedNames      map[string]bool                     // Map storing the names that are already assigned to tables, indices or foreign key contraints.
 	dataSink       func(table string, cols []string, values []interface{})
+	DataFlush      func()         `json:"-"`
```

Comment:
What does this struct tag do? Maybe add a comment next to it explaining the field?